protocols/imap/src/main/java/org/apache/james/imap/processor/SetACLProcessor.java

```diff
@@ -148,10 +147,7 @@ else if (!mailboxManager.hasRight(mailboxPath, Rfc4314Rights.a_Administer_RIGHT,
         } catch (MailboxNotFoundException e) {
             no(command, tag, responder, HumanReadableText.MAILBOX_NOT_FOUND);
         } catch (MailboxException e) {
-            Logger log = session.getLog();
-            if (log.isInfoEnabled()) {
-                log.info(command.getName() +" failed for mailbox " + mailboxName, e);
-            }
+            session.getLog().error(command.getName() +" failed for mailbox " + mailboxName, e);
```

Comment:
s/+"/+ "/

if you have time of course that is not really important. But my eyes don't like it